rollup.config.js

```diff
@@ -1,15 +1,17 @@
 import commonjs from '@rollup/plugin-commonjs'
 import { babel } from '@rollup/plugin-babel'
+import typescript from 'rollup-plugin-typescript2'
 
 export default {
-  input: './src/index.js',
+  input: './src/index.ts',
   output: [{
     file: './bin/index.js',
     format: 'cjs',
     banner: '#! /usr/bin/env node\n',
     exports: 'none'
   }],
   plugins: [
+    typescript(),
     babel({ babelHelpers: 'bundled' }),
     commonjs()
```

Comment:
Is this needed, is it not handled by typescript? Also, I think it should be the first plugin, before babel at least.